lib/clearance/configuration.rb

```diff
@@ -93,6 +98,7 @@ def initialize
       @cookie_name = "remember_token"
       @httponly = false
       @mailer_sender = 'reply@example.com'
+      @password_reset_time_limit = 15.minutes
```

Comment:
@jessieay - I don't think I understand why this should be an `ENV` var. I also don't think it'll work as one either since the environment wouldn't understand `15.minutes`. Can you further elaborate?